lib/active_merchant/billing/gateways/paypal.rb

```diff
@@ -73,6 +73,11 @@ def build_sale_or_authorization_request(action, money, credit_card_or_referenced
         xml.target!
       end
 
+      def api_version(options)
+        return 214 if options.dig(:three_d_secure, :version) =~ /^2/
```

Comment:
- I do not have much context but I feel misleading that the `API_VERSION = '124'` is alone, it give the impression that is *the* version.
- [Paypal's doc](https://developer.paypal.com/docs/classic/paypal-payments-pro/integration-guide/3d-secure/#integration-steps) uses `214.0` with decimal. Any significance?
- the request does not change otherwise between the two versions we have?

Reply:
- added a new constant for the version
- i originally did not think it's significant because in the docs they say to set that field to `214` and then in the example they use `214.0`, but it's better to pass whatever's used in the example. so thank you, i changed that
- as far as i can tell, request does not change, but it's required to add more fields if 3DS 1.x and even more fields if 2.x